docs/mobile-apps/automated-testing/appium/appium-2-deque-drivers.md

```diff
@@ -0,0 +1,183 @@
+  'appium:autoGrantPermissions': true,
+  'appium:autoAcceptAlerts': true,
+  'appium:newCommandTimeout': 90,
+  'sauce:options': {
+    name: "Android Driver - Testing appium2-deque-accessibility",
+    appiumVersion: 'appium2-deque-accessibility',
+  },
+}
+```
+</TabItem>
+<TabItem value="AxeXCUITEST">
+
+<!-- prettier-ignore -->
+```js
+const capabilities = {
+  platformName: 'iOS',
+  'appium:platformVersion': '^1(7|8).*$', // The iOS driver will work only on devices running iOS 17 and above
+  'appium:automationName': 'AxeXCUITEST', // New Automation name goes here
+  'appium:app': 'storage:filename=XXXXXXXXXx',
+  'appium:appPackage': 'xxxxxxxxxx',
+  'appium:autoGrantPermissions': true,
+  'appium:autoAcceptAlerts': true,
+  'appium:newCommandTimeout': 90,
+  'sauce:options': {
+    name: "iOS Driver - Testing appium2-deque-accessibility",
+    resigningEnabled: true,
+    appiumVersion: 'appium2-deque-accessibility',
+  },
+}
+```
+</TabItem>
+
+</Tabs>
+
+2. Obtain Your Deque API Key
+Log in to your [axe Deque account setting page](https://axe.deque.com/settings) and retrieve your API key. 
+This key is required to authenticate and initiate accessibility scans during your tests.
+
+3. Prepare Your Testing Script
+Once you have your API key, include it in your test script to perform a scan using the `mobile: axeScan` command. Below is an example script:
+
+```js
+describe("Test appium2-deque-accessibility - Using Drivers Android/iOS", () => {
+    it('test appium2-deque-accessibility', async () => {
+        const apiKey = process.env.DEQUE_API_KEY;
+        const settings = { apiKey: apiKey }
+
+        await driver.execute('mobile: axeScan', settings);
+    });
+});
+```
+## Migrating from the Deprecated Plugin to Deque New Drivers
+
+Deque's axeDevToolsMobile Appium Plugin is deprecated and will no longer be supported after January 31st 2025. Moving forward, 
+the following drivers should be used to ensure compatibility and continued support for accessibility testing:
+
+- Android: axe-appium-uiautomator2-driver: Automation name is `AxeUIAutomator2`
+- iOS : axe-appium-xcuitest-driver: Automation name is `AxeXCUITEST`
+
+### Why Migrate?
+- Continued Support: The plugin will no longer receive updates, fixes, or enhancements.
+- Expanded Capabilities: The drivers offer better integration and updated functionalities for Android and iOS platforms.
+- Compliance with Modern Requirements: For iOS, the new driver supports devices running iOS 17 and above.
+
+### Migration Steps
+1. ****Update Your Appium Capabilities****
+
+Replace the old plugin reference with the appropriate new driver. Choose the correct automationName based on your platform:
+
+Use `AxeUIAutomator2` for Android.
+Use `AxeXCUITEST` for iOS.
+
+Additionally, specify the appiumVersion as `appium2-deque-accessibility`.
+
+```js
+const capabilities = {
+  ...,
+  'appium:automationName': 'AxeUIAutomator2', // For Android; use 'AxeXCUITEST' for iOS
```

Comment:
It would be nice to have this with an Android and iOS tab and the code example.

Reply:
Thank, will update the MR